docs/source/markdown/podman-network-rm.1.md

```diff
@@ -21,15 +21,13 @@ Seconds to wait before forcibly stopping the running containers that are using t
 
 ## EXAMPLE
 
-Delete the `podman9` network
-
+Delete specified network:
 ```
 # podman network rm podman9
 Deleted: podman9
 ```
 
-Delete the `fred` network and all containers associated with the network.
-
+Delete the specified network and all containers associated with the network:
```

Comment:
Here we use an article but above we don't add a "the" before "specified".

Reply:
Removed.